### DISCUSSION TITLE
Impossible to create long audiobooks

### DISCUSSION THREAD

**Discussion by Artem54545** [yermak/AudioBookConverter#372] [General]
Hi, I wanted to merge 243 files, the total duration of which exceeded 24h. The programme stuck at the very end. Time kept going, but nothing was happening.  What is the largest number of files, that can be combined? What is the longest possible duration of an audiobook?

**Comment by yermak** [comment]
Hi, chapter numbering is limited with 3 digits, so more 1000 files may create a mess with chapter naming, unless you change pattern in configuration (currently in db).
I need to double check if there is problem with chapter duration marks - as they are in time format, potentially duration over 24 could impact it.
There should be no limit on file number or duration, so most likely it's broken chapters timestamps - I will test it in a few days.
